lake/oneshot_cls.py

```diff
@@ -232,7 +231,15 @@ def main():
     # Study
     # --------------------------------------------------------------------------
     for step in range(config['study_steps']):
-      model(study_data, study_target, mode='study')
+      study_train_losses, _ = model(study_data, study_target, mode='study')
+      study_train_loss = study_train_losses['stm']['memory']['loss']
+
+      print('Losses step {}, ite {}: \t PR:{:.6f}\
+      PR mismatch: {:.6f} \t PM-EC: {:.6f}'.format(idx, step,
+                                                      study_train_loss['pr'].item(),
+                                                      study_train_loss['pr_mismatch'].item(),
+                                                      study_train_loss['pm_ec'].item()))
+      model(recall_data, recall_target, mode='study_validate')
```

Comment:
Have you confirmed that the validation losses get logged in TensorBoard as well?

Reply:
Yes, they are shown in TensorBoard as well.
<img width="1142" alt="Screen Shot 2021-07-10 at 2 58 27 pm" src="https://user-images.githubusercontent.com/81640310/125152344-56eede00-e18f-11eb-82b0-17085f965f4c.png">